lib/guessingGameMakeGuess.js

```diff
@@ -2,7 +2,16 @@
  * Given a game and guess,
  */
 function guessingGameMakeGuess(game, guess) {
-
+  game.numGuesses++;
+  if (game.targetWord.includes(guess)) {
+      for (let i = 0; i < game.targetWord.length; i++){
+          if (game.targetWord[i] === guess){
+              let array = game.currentWord.split('');
```

Comment:
Don't use a generic name like `array` for a variable name when you can avoid it.

This works, but what you're doing is creating a new array on every iteration of the loop, modifying one element, joining it, and then discarding it.

Instead:

```javascript
let currentLetters = game.currentWord.split('');

for (let i = 0; i < game.targetWord.length; i++) {
  if (game.targetWord[i] === guess) {
    currentLetters[i] = guess;
  }
}

game.currentWord = currentLetters.join('');
```